src/Mod/Arch/ArchBuildingPart.py

```diff
@@ -567,10 +569,10 @@ def setProperties(self,vobj):
             vobj.ShowLabel = True
         if not "FontName" in pl:
             vobj.addProperty("App::PropertyFont","FontName","BuildingPart",QT_TRANSLATE_NOOP("App::Property","The font to be used for texts"))
-            vobj.FontName = Draft.getParam("textfont","Arial")
+            vobj.FontName = params.get_param("textfont")
         if not "FontSize" in pl:
             vobj.addProperty("App::PropertyLength","FontSize","BuildingPart",QT_TRANSLATE_NOOP("App::Property","The font size of texts"))
-            vobj.FontSize = Draft.getParam("textheight",2.0)
+            vobj.FontSize = params.get_param("textheight")
```

Comment:
Again here, different default from above...

Reply:
2mm is very small for an object in a building model. In both this case and the Arch Axis case, multiplying the `textheight` with the `DefaultAnnoScaleMultiplier` would be a good idea I think. That way these texts will have the same initial height as those created with Draft_Text.

For the future we should perhaps consider adding a `ScaleMultiplier` property  to these objects. I am not fully aware how the texts in a BuildingPart are used, but axis bubbles are annotations.